### PR TITLE
docs: qwiksand-box

### PR DESCRIPTION
**Pull request by manucorporat**
https://twitter.com/erikras/status/1666361545629024256

Screenshot 2023-06-07 at 14 44 40

### CODE REVIEW

**Review by erikras — APPROVED** [review]
👏 

giphy-25

File: packages/docs/src/components/header/header.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export const Header = component$(() => {
                 href="/examples/introduction/hello-world/"
                 class={{ active: pathname.startsWith('/examples') }}
               >
-                <span>Playground</span>
+                <span aria-aria-label='Qwik playground'>Qwik-sandbox</span>
```
[the remark this answers is not in the document]
Comment (bolonio) [reply packages/docs/src/components/header/header.tsx:104]: You don't really need the `aria-label` attribute, and definitely not in a `<span>` tag. If you ever need to add it, please add it to the interactive element. I've kept the suggestion from @erikras 

```suggestion
                <span>Qwik Sandbox</span>
```

[the remark this answers is not in the document]
Comment (erikras) [reply packages/docs/src/components/header/header.tsx]: Sorry. Was a copy-paste mistake.

File: packages/docs/src/components/header/header.tsx [hunk 2]
```diff
@@ -101,7 +101,9 @@ export const Header = component$(() => {
                 href="/examples/introduction/hello-world/"
                 class={{ active: pathname.startsWith('/examples') }}
               >
-                <span>Playground</span>
+                <span aria-aria-label="Qwik playground" class="qwiksand">
```
Comment (bolonio) [inline packages/docs/src/components/header/header.tsx:104]:
```suggestion
                <span class="qwiksand">
```

File: packages/docs/src/components/header/header.tsx [hunk 3]
```diff
@@ -101,7 +101,9 @@ export const Header = component$(() => {
                 href="/examples/introduction/hello-world/"
                 class={{ active: pathname.startsWith('/examples') }}
```
Comment (bolonio) [inline packages/docs/src/components/header/header.tsx:102]:
```suggestion
                class={{ active: pathname.startsWith('/examples') }}
                aria-label="Qwik playground" 
```

**Review by bolonio — APPROVED** [review]

File: packages/docs/src/components/header/header.tsx (hunk 3 above)
Comment (bolonio) [reply packages/docs/src/components/header/header.tsx:102]: I would stick with @erikras suggestion and remove directly the dash from the title, but it you still want to keep, please add the `aria-label` to the interactive element

**Comment by manucorporat** [comment]
Thank you very much Adrian! i think we might not need any label as i named it "qwik sandbox" as before it was "qwik-sand box" which was confusing.

learnt new things today :)

**Pull request merged**